src/map/MapRoutePath.js

```diff
@@ -1,27 +1,13 @@
 import { useTheme } from '@mui/styles';
 import { useId, useEffect } from 'react';
-import { useSelector } from 'react-redux';
 import { map } from './core/MapView';
+import { getSpeedColor } from '../common/util/colors';
 
-const MapRoutePath = ({ name, positions, coordinates }) => {
+const MapRoutePath = ({ positions }) => {
   const id = useId();
 
   const theme = useTheme();
 
-  const reportColor = useSelector((state) => {
```

Comment:
Why are we removing this?

Reply:
the color is based on the position speed

Reply:
Yeah, but it means you're ignoring the settings.